apps/prairielearn/src/lib/externalGraderResults.js

```diff
@@ -129,8 +130,11 @@ async function loadQueueUrl(sqs) {
 }
 
 async function processMessage(data) {
-  const jobId = Number.parseInt(data.jobId);
-  if (Number.isNaN(jobId)) {
+  /** @type {string} */
+  let jobId;
+  try {
+    jobId = IdSchema.parse(data.jobId);
+  } catch (err) {
```

Comment:
This was necessary to ensure jobId is a string when calling the newly-typed function, but I replaced it with parsing via IdSchema to keep the functionality.